FileWritingObjects/FileWritingObjects/NumSet.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace FileWritingObjects
+{
+    public class NumSet
+    {
+        public LinkedList<int> Set;
```

Comment:
You could make this a hash set for simplicity. Also, this should be private and can be made readonly.
```suggestion
        private readonly HashSet<int> _Set;
```